Documentation/contributing/testing/e2e.rst

```diff
@@ -6,728 +6,162 @@
+``https://github.com/cilium/cilium-cli``, and then running ``make install``.
 
-    INTEGRATION_TESTS=true ginkgo --focus="Runtime"
-
-Ginkgo searches for all tests in all subdirectories that are "named" beginning
-with the string "Runtime" and contain any characters after it. For instance,
-here is an example showing what tests will be ran using Ginkgo's dryRun option:
+Next, you need a Kubernetes cluster to run Cilium. The easiest way to create one
+is to use `kind <https://github.com/kubernetes-sigs/kind>`_. Cilium provides
+a wrapper script which simplifies creating K8s cluster with ``kind``. For example,
+to create a cluster consisting of 1 control-plane node, 3 worker nodes, without
+kube-proxy, and with ``DualStack`` enabled:
 
 .. code-block:: shell-session
 
-    $ INTEGRATION_TESTS=true ginkgo --focus="Runtime" -dryRun
-    Running Suite: runtime
-    ======================
-    Random Seed: 1516125117
-    Will run 42 of 164 specs
-    ................
-    RuntimePolicyEnforcement Policy Enforcement Always
-      Always to Never with policy
-      /Users/ianvernon/go/src/github.com/cilium/cilium/test/runtime/Policies.go:258
-    •
-    ------------------------------
-    RuntimePolicyEnforcement Policy Enforcement Always
-      Always to Never without policy
-      /Users/ianvernon/go/src/github.com/cilium/cilium/test/runtime/Policies.go:293
-    •
-    ------------------------------
-    RuntimePolicyEnforcement Policy Enforcement Never
-      Container creation
-      /Users/ianvernon/go/src/github.com/cilium/cilium/test/runtime/Policies.go:332
-    •
-    ------------------------------
-    RuntimePolicyEnforcement Policy Enforcement Never
-      Never to default with policy
-      /Users/ianvernon/go/src/github.com/cilium/cilium/test/runtime/Policies.go:349
-    .................
-    Ran 42 of 164 Specs in 0.002 seconds
-    SUCCESS! -- 0 Passed | 0 Failed | 0 Pending | 122 Skipped PASS
-
-    Ginkgo ran 1 suite in 1.830262168s
-    Test Suite Passed
-
-The output has been truncated. For more information about this functionality,
-consult the aforementioned Ginkgo documentation.
-
-.. _running_k8s_tests:
-
-Running Kubernetes Tests
-^^^^^^^^^^^^^^^^^^^^^^^^
-
-To run all of the Kubernetes tests, run the following command from the ``test`` directory:
-
-.. code-block:: shell-session
+    $ cd cilium/
+    $ ./contrib/scripts/kind.sh "" 3 "" "" "none" "dual"
+    ...
+    Kind is up! Time to install cilium:
+    make kind-image
+    make kind-install-cilium
 
-    INTEGRATION_TESTS=true ginkgo --focus="K8s"
+Afterwards, you need to install Cilium. The preferred way is to use
+`cilium-cli install <https://github.com/cilium/cilium-cli/#install-cilium>`_,
+as it is able to automate some steps (e.g., detecting ``kube-apiserver`` endpoint
+address which otherwise needs to be specified when running w/o ``kube-proxy``, or
+set an annotation to a K8s worker node to prevent Cilium from being scheduled on it).
```

Comment:
Clarity, no Latin abbreviations
```suggestion
After installing kind, the next step is to install Cilium. The preferred installation method is to use
`cilium-cli install <https://github.com/cilium/cilium-cli/#install-cilium>`_,
which can automate some steps (for example, detecting a ``kube-apiserver`` endpoint
address which otherwise needs to be specified when running without ``kube-proxy``; or
setting an annotation to a Kubernetes worker node to prevent Cilium from being scheduled on it).
```